admin_manual/installation/source_installation.rst

```diff
@@ -438,23 +440,18 @@ Then enable the newly created site::
 
 On CentOS/RHEL, create a virtualhost :file:`/etc/httpd/conf.d/nextcloud.conf`and add the following content to it:
```

Comment:
I guess the following block should be a code block. To trigger this `::` is needed like that:

```suggestion
On CentOS/RHEL, create a virtualhost :file:`/etc/httpd/conf.d/nextcloud.conf`and add the following content to it::
```